themes/uncontained.io/layouts/partials/sidebar.html

```diff
@@ -0,0 +1,14 @@
+	<aside>
+		<div>
+			<div>
+				<h3>Latest Posts</h3>
+			</div>
+			<div>
+				<ul>
+					{{ range first 5 .Site.Pages }}
```

Comment:
I think we need to target specific sections with this sidebar layout. Right now I get a list of pages that includes:

Home
Content Types
Guides
Articles
Openshift LDAP Integration

Reply:
Yeah, this feature is kind of weird - it came as part of the starter theme and isn't super helpful as-is. I'd propose removing the "sidebar" for now, and then adding more useful navigation in the future. Sound ok?

Reply:
Cool! I've made that change.